testutil/ioutil.go

```diff
@@ -65,7 +65,7 @@ func ApplyMockIODiscardOutErr(c *cobra.Command) BufferReader {
 // the caller must call to remove the file when it is
 // no longer needed.
 func WriteToNewTempFile(t testing.TB, s string) (*os.File, func()) {
-	fp, err := ioutil.TempFile("", t.Name()+"_")
+	fp, err := ioutil.TempFile("", strings.ReplaceAll(t.Name(), "/", "_")+"_")
```

Comment:
You should use [filepath.FromSlash()](https://golang.org/pkg/path/filepath/#FromSlash)